These were removed in libxkbcommon-1.6.0[1] and were apparently
unnecesary. Usage results in build failure.

Just a quick fix while waiting for what [2] comes up with.

Update: will be fixed by [3], see also [4]

[1] https://gitlab.freedesktop.org/xorg/proto/xorgproto/-/merge_requests/70
[2] https://bugreports.qt.io/browse/QTBUG-117950
[3] https://invent.kde.org/qt/qt/qtbase/-/merge_requests/290
[4] https://bugs.gentoo.org/915510
--- a/src/platformsupport/input/xkbcommon/qxkbcommon.cpp
+++ b/src/platformsupport/input/xkbcommon/qxkbcommon.cpp
@@ -276,4 +276,4 @@
-        Xkb2Qt<XKB_KEY_dead_lowline,            Qt::Key_Dead_Lowline>,
-        Xkb2Qt<XKB_KEY_dead_aboveverticalline,  Qt::Key_Dead_Aboveverticalline>,
-        Xkb2Qt<XKB_KEY_dead_belowverticalline,  Qt::Key_Dead_Belowverticalline>,
-        Xkb2Qt<XKB_KEY_dead_longsolidusoverlay, Qt::Key_Dead_Longsolidusoverlay>,
+        Xkb2Qt<0xfe90,                          Qt::Key_Dead_Lowline>,
+        Xkb2Qt<0xfe91,                          Qt::Key_Dead_Aboveverticalline>,
+        Xkb2Qt<0xfe92,                          Qt::Key_Dead_Belowverticalline>,
+        Xkb2Qt<0xfe93,                          Qt::Key_Dead_Longsolidusoverlay>,
